### PR TITLE
update WAGMI RPC URL

### PR DESCRIPTION
**Pull request by Julian-dev28**
Updated RPC URL to the most current endpoint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -670,12 +670,12 @@ Target Block Rate: 2s (Same as C-Chain)
 
 ```
 Network Name: WAGMI
-RPC URL: https://api-wagmi.avax-test.network/rpc
+RPC URL: https://subnets.avax.network/wagmi/wagmi-chain-testnet/rpc
 Chain ID: 11111
 Symbol: WGM
 ```
 
-![metamask](./imgs/metamask.png)
```
Comment (patrick-ogrady) [inline README.md:678]: The block explorer in this image should be: https://subnets.avax.network/wagmi/wagmi-chain-testnet/explorer

**Review by patrick-ogrady — CHANGES_REQUESTED** [review]
image is incorrect...should update explorer.

**Comment by Julian-dev28** [comment]
@patrick-ogrady 

Should be good to go now

**Review by Julian-dev28 — COMMENTED** [review]
@patrick-ogrady 

Good to go!

**Review by ceyonur — APPROVED** [review]
LGTM

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**